<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>webQQ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logindialog.qrc">
    <normaloff>:/login/Resources/QQ.png</normaloff>:/login/Resources/QQ.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/login/Resources/background2.png);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_Login">
    <widget class="QLabel" name="label_Login">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe Arabic</family>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#00ff00;&quot;&gt;登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Username">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>162</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_QQNumber</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_Password">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>192</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>密  码：</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_Password</cstring>
     </property>
    </widget>
    <widget class="MyLineEdit" name="lineEdit_QQNumber">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>162</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="maxLength">
      <number>11</number>
     </property>
    </widget>
    <widget class="MyLineEdit" name="lineEdit_Password">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>192</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Login">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/login/Resources/button_login_down.png);</string>
     </property>
     <property name="text">
      <string>登   录</string>
     </property>
     <property name="iconSize">
      <size>
       <width>160</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Status">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>345</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <item>
      <property name="text">
       <string>我在线上</string>
      </property>
      <property name="icon">
       <iconset resource="logindialog.qrc">
        <normaloff>:/status/Resources/Status/imonline.png</normaloff>:/status/Resources/Status/imonline.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Q我吧</string>
      </property>
      <property name="icon">
       <iconset resource="logindialog.qrc">
        <normaloff>:/status/Resources/Status/Qme.png</normaloff>:/status/Resources/Status/Qme.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>离开</string>
      </property>
      <property name="icon">
       <iconset resource="logindialog.qrc">
        <normaloff>:/status/Resources/Status/away.png</normaloff>:/status/Resources/Status/away.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>忙碌</string>
      </property>
      <property name="icon">
       <iconset resource="logindialog.qrc">
        <normaloff>:/status/Resources/Status/busy.png</normaloff>:/status/Resources/Status/busy.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>请勿打扰</string>
      </property>
      <property name="icon">
       <iconset resource="logindialog.qrc">
        <normaloff>:/status/Resources/Status/mute.png</normaloff>:/status/Resources/Status/mute.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>隐身</string>
      </property>
      <property name="icon">
       <iconset resource="logindialog.qrc">
        <normaloff>:/status/Resources/Status/invisible.png</normaloff>:/status/Resources/Status/invisible.png</iconset>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="widget_VerifyCode" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>400</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>48</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();</string>
      </property>
      <property name="text">
       <string>验证码：</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_Verifycode</cstring>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Verifycode">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();</string>
      </property>
     </widget>
     <widget class="verifyLabel" name="label_imgverify">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="verifyLabel" name="label_ChangeImg">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url();</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#ff0000;&quot;&gt;换一张&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_show">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>139</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_Setting">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>登录设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>网络设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>91</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;类型：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>91</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;地址：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>91</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;端口：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ProxyType">
     <property name="geometry">
      <rect>
       <x>83</x>
       <y>91</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <item>
      <property name="text">
       <string>不使用代理</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HTTP代理</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SOCKS5代理</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>使用浏览器设置</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ProxyAddress">
     <property name="geometry">
      <rect>
       <x>237</x>
       <y>91</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit__ProxyPort">
     <property name="geometry">
      <rect>
       <x>339</x>
       <y>91</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>117</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;用户名：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>117</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;密码：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>298</x>
       <y>117</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;域：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ProxyUsername">
     <property name="geometry">
      <rect>
       <x>83</x>
       <y>117</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ProxyPassword">
     <property name="geometry">
      <rect>
       <x>237</x>
       <y>117</y>
       <width>55</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ProxyYu">
     <property name="geometry">
      <rect>
       <x>339</x>
       <y>117</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ProxyTest">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>150</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/login/Resources/button_login_down.png);</string>
     </property>
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>登录服务器</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Right">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/login/Resources/button_login_down.png);</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Cancel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/login/Resources/button_login_down.png);</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>299</x>
       <y>210</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;端口：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ServerType">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>210</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <item>
      <property name="text">
       <string>不使用高级选项</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UDP类型</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP类型</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ServerPort">
     <property name="geometry">
      <rect>
       <x>338</x>
       <y>210</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>194</x>
       <y>210</y>
       <width>36</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;地址：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;类型：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ServerAddress">
     <property name="geometry">
      <rect>
       <x>235</x>
       <y>210</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url();</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Minisize">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/systoolbar/Resources/SysToolBar/sys_mini.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Setting">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/systoolbar/Resources/SysToolBar/sys_setting.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/systoolbar/Resources/SysToolBar/sys_close.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_QQ">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/login/Resources/QQ.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
   <slots>
    <signal>focusOutSignal()</signal>
    <signal>focusInSignal()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>verifyLabel</class>
   <extends>QLabel</extends>
   <header>verifylabel.h</header>
   <slots>
    <signal>Clicked(verifyLabel*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_QQNumber</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>lineEdit_Verifycode</tabstop>
  <tabstop>pushButton_Login</tabstop>
  <tabstop>comboBox_Status</tabstop>
  <tabstop>pushButton_Setting</tabstop>
  <tabstop>pushButton_Minisize</tabstop>
  <tabstop>pushButton_Close</tabstop>
  <tabstop>lineEdit_ProxyAddress</tabstop>
  <tabstop>lineEdit__ProxyPort</tabstop>
  <tabstop>lineEdit_ProxyUsername</tabstop>
  <tabstop>comboBox_ProxyType</tabstop>
  <tabstop>lineEdit_ProxyPassword</tabstop>
  <tabstop>lineEdit_ProxyYu</tabstop>
  <tabstop>pushButton_ProxyTest</tabstop>
  <tabstop>pushButton_Right</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
  <tabstop>comboBox_ServerType</tabstop>
  <tabstop>lineEdit_ServerPort</tabstop>
  <tabstop>comboBox_ServerAddress</tabstop>
 </tabstops>
 <resources>
  <include location="logindialog.qrc"/>
 </resources>
 <connections/>
</ui>
